### ISSUE TITLE
Does pt support qemu user mode? -- Cannot get qemu-system pid

### ISSUE THREAD

**Issue by lonnywong** [martinradev/gdb-pt-dump#18]
```sh
$ qemu-aarch64 -g 1234 -L /usr/aarch64-linux-gnu ./chall
```

```
$ gdb-multiarch -ex 'target remote 127.0.0.1:1234' ./chall

> pt -save -filter
Cannot get qemu-system pid
```

**Comment by martinradev** [comment]
@lonnywong 

I think qemu-user is for emulating userspace processes, and not a full aarch64 system. There is no benefit of using gdb-pt-dump for this usecase because there are no page tables exposed by qemu.

I will close this later this week unless you have some more queries.

**Comment by lonnywong** [comment]
@martinradev Can you parse it from `/proc/<pid>/maps` ?

**Comment by martinradev** [comment]
No, this project is not trying to solve this problem, and maps doesn't actually show the mapped pages, it shows the VMAs.

**Comment by lonnywong** [comment]
It includes the VMAs and the mapped pages. May need some translation.
